src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java

```diff
@@ -113,7 +111,7 @@ private void complete(IMessageReceiver receiver, IMessage msg) {
         } catch (InterruptedException | ServiceBusException e) {
             insights.trackMessageCompletedInServiceBus(Duration.between(start, Instant.now()), false);
 
-            logger.error("Unable to mark message " + msg.getMessageId() + " as processed");
+            logger.error("Unable to mark message {} as processed", msg.getMessageId());
```

Comment:
Doesn't sonar complain about not including the exception in the log?

Reply:
Not that I am aware of.It does complain if you do not include any parameters in log statement it does gives you a violation which is a FP and it is a bug in sonar.

Reply:
It's not strictly related with this pull request, but since you're changing the logging here, it would be worth including the exception, so that we would know what caused the problem.